app/lua/ltable.c

```diff
@@ -739,11 +739,14 @@ int luaH_getn (Table *t) {
 
 /* same thing for rotables */
 int luaH_getn_ro (void *t) {
-  int i = 1, len=0;
-
+  int len = 0;
+#if 0
```

Comment:
Some love wanted here, I think.

Reply:
So previous comments about whether to retain numeric indexes for ROTables.